databricks/koalas/tests/test_dataframe.py

```diff
@@ -581,6 +581,132 @@ def mul10(x) -> int:
         kdf5 = kdf3 + 1
         self.assert_eq(kdf5.rename(index=str_lower), pdf5.rename(index=str_lower))
 
+    def test_rename_axis(self):
+        index = pd.Index(["A", "B", "C"], name="index")
+        columns = pd.Index(["numbers", "values"], name="cols")
+        pdf = pd.DataFrame([[1.0, 2.0], [3.0, 4.0], [5.0, 6.0]], index=index, columns=columns)
+        kdf = ks.from_pandas(pdf)
+
+        for axis in [0, "index"]:
+            self.assert_eq(
+                pdf.rename_axis("index2", axis=axis).sort_index(),
+                kdf.rename_axis("index2", axis=axis).sort_index(),
+            )
+
+        for axis in [1, "columns"]:
+            self.assert_eq(
+                pdf.rename_axis("cols2", axis=axis).sort_index(),
+                kdf.rename_axis("cols2", axis=axis).sort_index(),
+            )
+
+        self.assert_eq(
+            pdf.rename_axis(["index2"]).sort_index(), kdf.rename_axis(["index2"]).sort_index(),
+        )
+
+        self.assert_eq(
+            pdf.rename_axis(["index2"], axis=1).sort_index(),
+            kdf.rename_axis(["index2"], axis=1).sort_index(),
+        )
+
+        self.assertRaises(ValueError, lambda: kdf.rename_axis(["index2", "index3"], axis=0))
+        self.assertRaises(ValueError, lambda: kdf.rename_axis(["cols2", "cols3"], axis=1))
+
+        # index/columns parameters and dict_like/functions mappers introduced in pandas 0.24.0
+        if LooseVersion(pd.__version__) >= LooseVersion("0.24.0"):
```

Comment:
Would you make tests for `pandas < 0.24.0` by manually declaring the expected result like we did before ??